python/katana/analytics/_bfs.pyx

```diff
@@ -190,7 +190,7 @@ def bfs_assert_valid(PropertyGraph pg, str property_name):
     output_property_name_bytes = bytes(property_name, "utf-8")
     output_property_name_cstr = <string>output_property_name_bytes
     with nogil:
-        handle_result_assert(BfsAssertValid(pg.underlying_property_graph(), output_property_name_cstr))
+        handle_result_assert(BfsAssertValid(pg.underlying_property_graph(), start_node, output_property_name_cstr))
```

Comment:
start_node isn't an argument to this function at the Cython level. You will need to add it above. And add it in the test cases.

Reply:
I'm fixing this. Commit in coming.